uint32 id
uint32 source_id
uint32 target_id
distributed_slam_karto/Link link